### ISSUE TITLE
Suggestions for Fine-Tuning?

### ISSUE THREAD

**Issue by PascalHbr** [clovaai/deep-text-recognition-benchmark#270]
I want to fine-tune the model on my own dataset. The default lr is set to 1., which seems to be quite high. What lr is suggested for fine-tuning? 

Moreover, is it recommended to fine-tune on cut out single digits/characters or on complete images? 

My last question is about the input parameters concerning the image width/height. The default parameters are 100/32. Is it recommended to adjust this during the fine-tuning step? My fine-tuning dataset is of size 64x64, should i adjust the parameters to 64x64 accordingly?

Thanks a lot in advance!

**Comment by hisiter97** [comment]
1. Learning rate for fine tuning can be 1e-4 
2.None
3. If you use pretrained model, you must use image size 32x32. If training from scratch, you can use 64x64

**Comment by mjack3** [comment]
@hisiter97 I think he can fine tunning with the widthxheight that he desire, no matter from scratch or fine tunning.
